api/src/services/users/login_gov_callback_handler.py

```diff
@@ -158,6 +159,23 @@ def _process_token(db_session: db.Session, token: str, nonce: str) -> LoginGovCa
     # NOTE: This doesn't commit yet - but effectively moves the cache from memory to the DB transaction
     db_session.flush()
 
+    # Check if the user is an ebiz POC and create/link their organization
+    # Only do this for new users
+    if is_user_new:
+        org_user = handle_ebiz_poc_organization_during_login(db_session, external_user.user)
+
+        if org_user:
+            logger.info(
+                "User linked to organization as organization owner (ebiz POC)",
+                extra={
+                    "user_id": str(external_user.user.user_id),
+                    "organization_id": str(org_user.organization_id),
+                    "is_organization_owner": org_user.is_organization_owner,
+                },
+            )
+            # Flush again to ensure organization data is persisted
+            db_session.flush()
```

Comment:
I'd actually suggest trimming most of this - the function we call logs the same thing, and I don't think the flush is needed (really the flush above also could be worked around too, not sure why I did it like that).
```suggestion
    if is_user_new:
        handle_ebiz_poc_organization_during_login(db_session, external_user.user)
```

Reply:
Also updated this, much cleaner.